Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +254,224 @@ edgebridgetutorialapp://
+
+After creating the desired mappings, select **Save** in the top right.
+
+### Mapping calculated fields using Analytics functions
+
+1. Select the **Add calculated field** button (**1**).
+
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-start-mapping-calculated.png" alt="Datastream start mapping calculated field" width="1100"/>  
+
+1. In the **Create Calculated Field** window, check that the **Function** tab (**1**) is selected, then narrow down to the Analytics functions by inputting `aa` into the search box (**2**).
+
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-create-calculated-field.png" alt="Datastream create calculated field" width="1100"/>  
+
+1. Select the plus button next to **aa_get_product_names** (**1**) and notice that the function is populated in the main editor window.
+2. Next, select the **Field** tab (**2**).
+
+> [!TIP]
+> Selecting the function name itself will display help text in the right-side help panel, which will give you more context on how to use the function.
 
-| JSON Property                           | XDM Property                       | trackAction        | trackState         |
-| --------------------------------------- | ---------------------------------- | ------------------ | ------------------ |
-| data.contextdata.product\\.add\\.event  | commerce.productListAdds.value     | :white_check_mark: |                    |
-| data.contextdata.product\\.view\\.event | commerce.productListViews.value    |                    | :white_check_mark: |
-| data.contextdata.product\\.id           | productListItems.SKU               | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.name         | productListItems.name              | :white_check_mark: | :white_check_mark: |
-| data.contextdata.product\\.units        | productListItems.quantity          | :white_check_mark: |                    |
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-calculated-field-set-function.png" alt="Datastream create calculated field" width="1100"/>  
 
-12. After completing all the mappings, click **Save**.
+1. Set the editor cursor between the parentheses of the `aa_get_product_names()` function in the editor area (**1**).
 
-## Final validation using Assurance
+<img src="../assets/edge-bridge-tutorial/data-prep/datastreams-calculated-field-pre-set-field.png" alt="Datastream create calculated field" width="1100"/>  
+
+1. Select the plus button (**+**) next to **data.__adobe.analytics.products** (**1**) - it may appear truncated like: `data.__adobe.a...products`
+   1. Notice the green checkmark (**2**) in the top right of the editor area indicating correct syntax.
+   2. Select **Preview** (**3**) to see what the output of the function will be, based on what is currently in the editor and the original source JSON.
```

Comment:
while this is great info I feel it is little too detailed and takes the focus off the tutorial steps, and small tweaks like these can change in the UI over time eventually making our content go stale. e.g. we can simplify to:

Select the plus button (+) next to data.__adobe.analytics.products (1). You may select Preview (3) to see what the output of the function will be.